src/DirectiveContract.php

```diff
@@ -0,0 +1,12 @@
+<?php declare(strict_types=1);
+
+namespace Mazedlx\FeaturePolicy;
+
+interface DirectiveContract
```

Comment:
I'm creating classes for the different directives, so I'm also adding a contract for those classes to keep it consistent.